warehouse/query-core/src/main/java/datawave/query/language/functions/jexl/UniqueBySecond.java

```diff
@@ -7,7 +7,7 @@
 import java.util.ArrayList;
 
 /**
- * Function to return a unique result for every month of the year for a given list of fields. This function is equivalent to {@code #unique(field[MONTH])}.
+ * Function to return a unique result for every second for a given list of fields. This function is equivalent to {@code #unique(field[MONTH])}.
```

Comment:
Should be `#unique(field[SECOND])`. There are similar updates to the other UniqueBy* classes.

Reply:
will fix